omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/SessionController1_9.java

```diff
@@ -139,12 +141,14 @@ protected Provider getCurrentProvider() {
 			Collection<Provider> providers = new HashSet<Provider>();
 			try {
 				Context.addProxyPrivilege(PrivilegeConstants.VIEW_PROVIDERS);
+				Context.addProxyPrivilege(GET_PROVIDERS_PRIVILEGE);
```

Comment:
Nit: also include the comment above here, as this is where people would most likely be scratching their heads?